src/common.jl

```diff
@@ -8,7 +8,8 @@ using
 
 using
     GLMakie,
-    GeoMakie
+    GeoMakie,
+    ColorSchemes
```

Comment:
Good that you grouped like imports, but Blue style also says to alphabetize them too